cmd/cron_tasks.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"github.com/Scalingo/cli/appdetect"
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/crontasks"
+	"github.com/urfave/cli"
+)
+
+var (
+	CronTasksCommand = cli.Command{
```

Comment:
Is there a reason for this variable to be public? And the name does not sound really good

```suggestion
	cronTasksListCommand = cli.Command{
```